Source/OxyPlot/Rendering/SimpleTextTrimmer.cs

```diff
@@ -0,0 +1,217 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="SimpleTextTrimmer.cs" company="OxyPlot">
+//   Copyright (c) 2020 OxyPlot contributors
+// </copyright>
+// <summary>
+//   A simple trimmer that doesn't use glyph information.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot.Rendering
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq.Expressions;
+    using System.Text.RegularExpressions;
+
+    /// <summary>
+    /// A simple trimmer that doesn't use glyph information.
+    /// </summary>
+    public class SimpleTextTrimmer : ITextTrimmer
+    {
+        /// <summary>
+        /// The default ellipsis, comprising three ascii stop symbols.
+        /// </summary>
+        public static readonly string AsciiEllipsis = "...";
+
+        /// <summary>
+        /// The unicode horizontal ellipsis.
+        /// </summary>
+        public static readonly string UnicodeEllipsis = "…";
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SimpleTextTrimmer" /> class.
+        /// </summary>
+        public SimpleTextTrimmer()
+        {
+            this.TrimToWord = false;
+            this.Ellipsis = AsciiEllipsis;
+            this.AppendEllipsis = false;
```

Comment:
Should we enable `AppendEllipsis` by default? Now that we have it in there, I think in most cases this would be preferable.
Also, I'm wondering whether the default ellipsis should be the `UnicodeEllipsis` - this will give the better appearance where supported, but could be a bit more problematic with font support.